platform/solana/client.go

```diff
@@ -0,0 +1,38 @@
+package solana
+
+import (
+	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+)
+
+const stakeProgramId  = "Stake11111111111111111111111111111111111111"
```

Comment:
just run go fmt ./...

Reply:
Fwiw, running that specific command reformatted coins/coins.go, which broke unit tests

Reply:
Also had the same issue sometimes :) 
We will try to solve it